docs/how-to/migrate-to-mas.md

```diff
@@ -0,0 +1,145 @@
+# How to back up and restore Synapse
+This document shows how to migrate a legacy synapse homeserver (`latest/edge` track) to an oidc-native homeserver (`2/edge` track).
```

Comment:
What about adding a warning to create a backup and mentioning that this procedure is irreversible? Also, the current instance should be stopped during the process.

Reply:
Hmm, I'm not sure that this process is irreversible as we don't touch the existing database, we can always remove the new charm and start over again.